haskell/private/ghci_repl_wrapper.sh

```diff
@@ -6,15 +6,34 @@
 # likely that the script is invoked incorrectly, so we need to output a
 # warning with instructions how to invoke it.
 
-GHCI_LOCATION="{GHCi}"
-SCRIPT_LOCATION="{SCRIPT_LOCATION}"
+if [ ! -d "bazel-out" ]; then
+  echo "The bazel-out symlink must be present in workspace directory."
```

Comment:
As this fix will stop working when bazelbuild/bazel #3796 will be fixed, perhaps an error message with more information can be useful. Especially I can imagine a user with an indirect bazel upgrade ending to this error message. Something such as `"The bazel-out symlink must be present in workspace directory. This is not the case and it means that you upgraded to a bazel version not (yet) supported by rules_haskell. Please report the issue to rules_haskell."